xline/src/server/kv_server.rs

```diff
@@ -140,7 +140,13 @@ where
         #[allow(clippy::wildcard_enum_match_arm)]
         if use_fast_path {
             let cmd_res = self.client.propose(cmd).await.map_err(|err| match err {
-                ProposeError::ExecutionError(e) => tonic::Status::invalid_argument(e),
+                // If an error occurs when `prepare` or `execute`, `after_sync` will not be called, in this case,
```

Comment:
This comment looks a little bit weird. Is it a grammar mistake? How about this?
```suggestion
                // If an error occurs during the `prepare` or `execute` stages, `after_sync` will not be invoked. In this case, 
                // `wait_synced` will return the errors generated in the first two stages. Therefore, 
                // if the response from `slow_round` arrives earlier than `fast_round`, the `propose` 
                // function will return a `SyncedError`, even though `after_sync` is not called.
```